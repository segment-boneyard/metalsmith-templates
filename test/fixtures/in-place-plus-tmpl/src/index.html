---
name: First Last
template: layout.html
---

{{name}}
