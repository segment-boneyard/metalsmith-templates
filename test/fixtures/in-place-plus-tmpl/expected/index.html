<html>
<body>
hello

First Last

</body>
</html>
